mirrord/operator/src/crd/kafka.rs

```diff
@@ -23,6 +24,11 @@ pub struct MirrordKafkaClientConfigSpec {
     ///
     /// The list of all available properties can be found [here](https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md).
     pub properties: Vec<MirrordKafkaClientProperty>,
+
+    /// Properties to set from a secret.
+    ///
+    /// The secret is resolved when resolving the rest of the final configuration.
+    pub load_from_secret: Option<Secret>,
```

Comment:
You can also add a `printcolumn` entry for this field (makes `kubectl get ...` output nicer)